### DISCUSSION TITLE
Intune App Configuration Profile for iOS

### DISCUSSION THREAD

**Discussion by dissidius** [rustdesk/rustdesk#4489] [Feature Request]
As in the title can it be added ? I mean for other OSs we can build with gitactions or build from source, while for ipa/iOS it is still not possibile so i was wondering if at least we coud get Intune App Configuration Profile to set RENDEZVOUS_SERVER and RS_PUB_KEY.

Cheers and thanks for the awesome work and effeort!

**Comment by rustdesk** [comment]
https://www.reddit.com/r/rustdesk/comments/13vtg29/intune_deployment_gather_passwords_set_passwords/
https://github.com/rustdesk/rustdesk/discussions/4332

We will follow up soon.
